rotkehlchen/db/evmtx.py

```diff
@@ -594,3 +581,27 @@ def get_or_create_genesis_transaction(
                     relevant_address=account,
                 )
         return tx
+
+    def _get_evm_transaction_query(self, query: str, has_premium: bool) -> str:
+        """Return queries for the evm_transaction database table"""
```

Comment:
Make the changes I asked above to this function, along with adjusting the docstring after the changes.